cypress/e2e/spec.cy.js

```diff
@@ -1,4 +1,4 @@
-import { absoluteRoot, without } from '../../src'
+import { absoluteRoot, without } from '../../index'
```

Comment:
Please revert this. Never import from parent index, it creates dependency loops when combined with re-exports.

Reply:
Never mind, this package has old structure.